### PR TITLE
Lab2

### PR DESCRIPTION
**Pull request by AndreyVanyuck**

### CODE REVIEW

File: app/src/main/java/com/example/timerppo/Activity/CreateWorkoutActivity.java [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+                            inputName.getText().toString(),
+                            Integer.parseInt(inputPrepair.getText().toString()),
+                            Integer.parseInt(inputWork.getText().toString()),
+                            Integer.parseInt(inputRest.getText().toString()),
+                            Integer.parseInt(inputCycle.getText().toString()),
+                            Integer.parseInt(inputSet.getText().toString()),
+                            Integer.parseInt(inputCalm.getText().toString()),
+                            Color.HSVToColor(new float[]{color.getFloatH(), color.getFloatL(), color.getFloatS()})
+                    );
+                    db.timerDao().insert(workoutModel);
```
Comment (gmltA) [inline app/src/main/java/com/example/timerppo/Activity/CreateWorkoutActivity.java]: All this code goes to ViewModel. You shouldn't interact with DB from your view layer.

File: app/src/main/java/com/example/timerppo/Activity/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        db = DatabaseHandler.getInstance().getDatabase();
+
+        listView = findViewById(R.id.listTimer);
+        CustomAdapter adapter = new CustomAdapter(this, R.layout.list_item, db.timerDao().getAll(), db);
```
Comment (gmltA) [inline app/src/main/java/com/example/timerppo/Activity/MainActivity.java]: Adaptor must not access DB.

**Review by gmltA — CHANGES_REQUESTED** [review]

**Review by gmltA — CHANGES_REQUESTED** [review]
If only I could compile this project... Gonna ask you to upload an APK I can run.

**Comment by AndreyVanyuck** [comment]
Added apk and screenshots

**Pull request closed without merging**